### PR TITLE
Devc

### PR DESCRIPTION
**Pull request by ctaylor451**

### CODE REVIEW

File: main/templates/event_detail.html [hunk 1]
```diff
@@ -269,7 +269,8 @@
                                 location.href='{% url "message_add" %}?eid={{ event.id }}' +
                                 '&OP={{ period.position }}' +
                                 '&period={{ period.id }}' +
-                                '&period_format=info';
+                                '&period_format=info'+
+                                '&page=Team msg';
```
Comment (kduncklee) [inline main/templates/event_detail.html]: Why do we need a template on these? It will create errors until we add it to the database. Let's discuss this and put it in a separate PR if we decide to add it.

File: main/templates/event_detail.html [hunk 2]
```diff
@@ -281,7 +282,8 @@
                                 location.href='{% url "message_add" %}?eid={{ event.id }}' +
                                 '&OP={{ period.position }}' +
                                 '&period={{ period.id }}' +
-                                '&period_format=broadcast';
+                                '&period_format=broadcast' +
+                                '&page=Unit msg';
```
Comment (kduncklee) [inline main/templates/event_detail.html]: Please remove this one too for now.

File: main/views/message_views.py [hunk 3]
```diff
@@ -72,7 +72,7 @@ def get_queryset(self):
                 period_format, self.request.body))
 
             rsvp_template = None
-            if page is not None:
+            if period_format not in ('info', 'broadcast'):
```
Comment (kduncklee) [inline main/views/message_views.py]: Why not keep this as is?

File: main/templates/event_detail.html (hunk 1 above)
Comment (ctaylor451) [reply main/templates/event_detail.html]: We discuss the different setting.  I don't know what you are using them for.  What is period_format?  Is it a template?  Is null OK?

Page is used to display the type of page being sent

File: main/views/message_views.py (hunk 3 above)
Comment (ctaylor451) [reply main/views/message_views.py]: Page is used to display the type of page being sent

File: main/templates/event_detail.html (hunk 2 above)
Comment (ctaylor451) [reply main/templates/event_detail.html]: see comment above

File: main/templates/event_detail.html (hunk 1 above)
Comment (kduncklee) [reply main/templates/event_detail.html]: In the backend, period_format is used to determine the type of message and how to handle it and its replies.

Page is used to pick the rsvp_template which is looked up in the database. This is used to determine the text appended in the UI, as well as the yes/no prompts in email. If rsvp_template is null, no response is expected.

It would be cleaner to rename page to rsvp_template so the names match and then you could add a new name if you need an additional field for UI elements.

Comment (ctaylor451) [reply main/templates/event_detail.html]: Why not merge period_format and page into a single concept, page_type. 

invite, heads up -> headsup
invite, available -> available
leave, Left -> leave
return, Returned -> return
 info, team msg -> null
broadcast, unit msg -> null
test, test -> null

Am I missing something?  Seems like anything with a reply requires special handling

Comment (ctaylor451) [reply main/templates/event_detail.html]: Rewrote code to clean this up.  Now uses page_format

File: main/views/message_views.py [hunk 4]
```diff
@@ -36,12 +36,21 @@ class MessageCreateView(LoginRequiredMixin, generic.ListView):
 
     def get_queryset(self):
         """Return context for standard paging."""
+        format_convert = {'headsUp': ['invite', 'Heads Up'],
+                          'available': ['invite', 'Available?'],
+                          'leave': ['leave', 'Left?'],
+                          'return': ['return', 'Returned?'],
+                          'info': ['info', None],
+                          'broadcast': ['broadcast', None],
+                          'test': ['test', 'test']
```
Comment (kduncklee) [inline main/views/message_views.py]: The rsvp template should be 'Test'.

**Comment by ctaylor451** [comment]
What's in the test rsvp template? I'll fix the name

File: main/views/message_views.py (hunk 4 above)
Comment (ctaylor451) [reply main/views/message_views.py]: Fixed
